documentation/library_guide/kernel_templates/kernel_configuration.rst

```diff
@@ -0,0 +1,39 @@
+Kernel Configuration
+####################
+
+.. code:: cpp
+
+   // Defined in header <oneapi/dpl/experimental/kernel_templates>
+
+   template <std::uint16_t DataPerWorkItem,
+             std::uint16_t WorkGroupSize,
+             typename KernelName = oneapi::dpl::execution::DefaultKernelName>
+   struct kernel_param;
+
+Each Kernel Template is supplied by a `kernel_param` object, generic structure for configuring a kernel.
+The supported and the most performant values depend on the Kernel Template itself, device capabilities, the data type and the data size.
+
+Member constants
+----------------
+
++------------------------------------------------------+---------------------+------------------------------------------------------+
+| Name                                                 | Value               | Description                                          |
++======================================================+=====================+======================================================+
+| ``static constexpr std::uint16_t data_per_workitem`` | ``DataPerWorkItem`` | Number of iterations to be processed by a work-item. |
```

Comment:
Work-item does not need a hyphen. Please use work item.

Reply:
Let's retain this spelling: `work-item` is a term from [SYCL™ 2020 Specification](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#glossary).

Reply:
Got it. I am fine with work-item in this case.